linkedevents/settings.py

```diff
@@ -55,16 +55,19 @@ def get_git_revision_hash() -> str:
     INTERNAL_IPS=(list, []),
     LANGUAGES=(list, ["fi", "sv", "en", "zh-hans", "ru", "ar"]),
     LIPPUPISTE_EVENT_API_URL=(str, None),
-    MAIL_MAILGUN_API=(str, ""),
-    MAIL_MAILGUN_DOMAIN=(str, ""),
-    MAIL_MAILGUN_KEY=(str, ""),
+    MAILGUN_API_URL=(str, ""),
+    MAILGUN_SENDER_DOMAIN=(str, ""),
+    MAILGUN_API_KEY=(str, ""),
     MEDIA_ROOT=(environ.Path(), root("media")),
     MEDIA_URL=(str, "/media/"),
     MEMCACHED_URL=(str, "127.0.0.1:11211"),
+    SEAT_RESERVATION_DURATION=(int, 15),
     SECRET_KEY=(str, ""),
     SECURE_PROXY_SSL_HEADER=(tuple, None),
     SENTRY_DSN=(str, ""),
     SENTRY_ENVIRONMENT=(str, "development"),
+    SOCIAL_AUTH_TUNNISTAMO_KEY="linkedevents-api-dev",
```

Comment:
This probably needs to be: 
```suggestion
    SOCIAL_AUTH_TUNNISTAMO_KEY=(str, "linkedevents-api-dev"),
```